src/examples/mailchimp/src/backend/schema/category/entity.ts

```diff
@@ -0,0 +1,13 @@
+import { Field, GraphQLEntity, ID, ObjectType, SummaryField } from '@exogee/graphweaver';
+
+import { CategoryDataEntity } from './data-entity';
+
+@ObjectType()
```

Comment:
Maybe just split these into 

/entities/category.ts
and
/schema/category.ts

?

This is a stylistic difference so it's up to you. I think we should be consistent with the other examples and keep data entities and schema entities in their own folder.

Reply:
Fixed as suggested.